runtime/Cpp/runtime/src/ParserRuleContext.h

```diff
@@ -1,11 +1,12 @@
-﻿/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
+/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
```

Comment:
FYI: This is the superfluous and Byte Order Mark, that was removed while editing. This is UTF-8, not UTF-16.